Respond to @orchestrator mention in #219

This task was created from an @orchestrator mention.

- **Repo:** `gabrielkoerich/orchestrator`
- **Target:** #219
- **Author:** @gabrielkoerich
- **Comment:** https://github.com/gabrielkoerich/orchestrator/issues/219#issuecomment-3940293913
- **Created:** 2026-02-22T06:39:29Z

### Mention Body

```markdown
Work complete for task #219.

**What I found:** Issue #202 was created to respond to `@orchestrator fix.` mentioned on PR #183 (feat(github): owner slash commands). A previous agent (codex) already completed the fix work:
- Diagnosed CI failures as shellcheck SC1072/SC1073 from unescaped markdown backticks in `scripts/lib.sh`  
- Opened PR #207 with the fix
- Replied on PR #183

**What I did:** Posted a response on issue #202 (https://github.com/gabrielkoerich/orchestrator/issues/202#issuecomment-3940292947) confirming:
- Both PR #206 and PR #207 have all CI checks passing ✅
- Recommended merging PR #207 (closes #202) and closing #206 as a duplicate

No code changes were needed — this was a status reporting task.
```

### Instructions

Respond back on #219 with your results and next steps.

--- a/docs/content/workflow.md
+++ b/docs/content/workflow.md
@@ -23,6 +23,20 @@ Issue → Branch + Worktree → Agent works → Push → PR → Review Agent →
 9. **Release** — CI auto-tags, generates changelog, creates GitHub release, updates Homebrew
 10. **Cleanup** — (TODO) orchestrator detects merged PR, removes worktree + local branch
 
+## Mention-Driven Tasks
+
+When someone comments `@orchestrator ...` on a GitHub issue/PR, the GitHub mentions listener can create a task like:
+
+```
+Respond to @orchestrator mention in #<N>
+```
+
+Expected outcome:
+
+- Read the mention body + any referenced issues/PRs
+- Reply back on the *target* issue with a concise status update and clear next steps
+- If no code/docs changes are required, the task can be completed without opening a PR
+
 ## Task Lifecycle
 
 ```

